kvirt/kvm/__init__.py

```diff
@@ -1561,13 +1561,17 @@ def create_network(self, name, cidr, dhcp=True, nat=True):
             natxml = "<forward mode='nat'><nat><port start='1024' end='65535'/></nat></forward>"
         else:
             natxml = ''
+        if domain:
```

Comment:
as domain defaults to '', this if will always evaluates to True
you rather want to default to None and use
if domain is not None: